pages/LearnerDashboard.html

```diff
@@ -0,0 +1,41 @@
+<section id="main_content" class="inner">
+  <h2>
+    <a id="Learner Dashboard" class="anchor"
+       href="#Learner_Dashboard" aria-hidden="true">
+      <span class="octicon octicon-link"></span>
+    </a>
+    Learner Dashboard
+  </h2>
+
+  <p>
+    Oppia's mission from the very start has been - "To help anyone learn anything they want in an effective and enjoyable way.". The "Learner Dashboard" has been designed and developed keeping this goal in mind. If you're a new user and you have signed in for the first time, here's how the learner dashboard would look -
+  </p>
+
+  <img src="static/images/noActivityLearnerDashboard.png" style="width: 100%;">
+
+  <p>
+    The page urges you to go out there and explore, learn new things! Once you start learning the page starts to look a bit different. It's primary goal is to track how much you've learnt which we believe is one of the crucial steps in learning. After all, if you don't know where you are, how will you get to where you want to be?
+  </p>
+
+  <p>
+    Here's how the learner dashboard looks to an experienced learner.
+  </p>
+
+  <img src="static/images/activeLearnerDashboard.png" style="width: 100%;">
+
+  <p>
+    As you can see, all the information about your progress is neatly organized in the learner dashboard. On the left hand side you can see a menu which tracks different things. There's the completed section which keeps track of all the explorations and collections you have completed. Then there's the 'In Progress' section which tracks of all the explorations and collections currently being completed. Information within these is further divided into explorations and collections which makes it easier to see how far you've gotten.
```

Comment:
Might be easier for the reader if we break these down into an unordered list.

On the left hand side you can see a menu which tracks different aspects of your Oppia experience:
```
<ul>
  <li><strong>In Progress</strong>: tracks all of the explorations and collections you are currently completing</li>
  <li><strong>Completed</strong>: tracks all of the explorations and collections you have completed</li>
  <li><strong>Subscriptions</strong>: tracks all of the creators to whom you have subscribed</li>
</ul>
```